src/title/TitleStream.js

```diff
@@ -18,17 +18,20 @@ export default class TitleStream extends React.PureComponent {
 
   props: {
     subscriptions: [],
+    streams: [],
     narrow: () => {},
     color: string,
   };
 
   render() {
-    const { narrow, subscriptions, color } = this.props;
-    const stream = subscriptions.find(x => x.name === narrow[0].operand) || {
-      name: '',
-      invite_only: false,
-      in_home_view: false
-    };
+    const { narrow, subscriptions, streams, color } = this.props;
+    const stream = subscriptions.find(x => x.name === narrow[0].operand) ||
+      { ...streams.find(x => x.name === narrow[0].operand), in_home_view: true } ||
```

Comment:
Yes streams contains all items, but it does not contain `in_home_view` which is used to determine stream icon [here](https://github.com/zulip/zulip-mobile/blob/master/src/title/TitleStream.js#L43)